src/Raven.Server/Documents/Indexes/Persistence/Corax/CoraxIndexingExtensions.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System;
+using Corax;
+
+namespace Raven.Server.Documents.Indexes.Persistence.Corax
+{
+    public class CoraxIndexingExtensions
+    {
+        public static Analyzer CreateAnalyzerInstance(string fieldName, Type analyzerType)
+        {
+            throw new NotImplementedException("Custom analyzers are not allowed in Corax.");
```

Comment:
Shoudn't we take advantage of `LuceneAnalyzerAdapter` at this point of the implementation? As I see we call this method from `CreateStandardAnalyzer` and `CreateKeywordAnalyzer`

Reply:
Done